packages/nodes-base/nodes/Todoist/v2/OperationHandler.ts

```diff
@@ -13,16 +24,20 @@ export interface OperationHandler {
 export interface CreateTaskRequest {
 	content?: string;
 	description?: string;
-	project_id?: number;
-	section_id?: number;
-	parent_id?: string;
+	project_id?: number | string;
+	section_id?: number | string;
+	parent_id?: number | string;
```

Comment:
api accepts either string or number. before the UI was passing strings but were casted "as number" here. 
since we did not enforce this before but now we validate this, I allowed both.